packages/list/test/mwc-list.test.ts

```diff
@@ -1788,5 +1788,29 @@ suite('mwc-list:', () => {
         }
       });
     });
+
+    suite('performance issue', () => {
+      test(
+          'removing a list should not call layout more than once', async () => {
+            let count = 0;
+            const originalLayout = List.prototype.layout;
+            List.prototype.layout = function(update) {
+              originalLayout.call(this, update);
+              count++;
+            };
+
+            const itemsTemplates = new Array(300).fill(0).map(() => listItem());
+            fixt = await fixture(listTemplate({items: itemsTemplates}));
+            element = fixt.root.querySelector('mwc-list')!;
+
+            count = 0;
+
+            fixt.remove();
+            await element.updateComplete;
+            fixt = null;
+            expect(count).to.eq(1);
```

Comment:
really dumb issue: please use `assert.equal` because internally we use Jasmine while out here we use karma and chai